dukan/src/main/kotlin/net/thechance/dukan/repository/DukanRepository.kt

```diff
@@ -158,4 +158,12 @@ interface DukanRepository : JpaRepository<Dukan, UUID> {
         @Param("dukanId") dukanId: UUID,
         @Param("status") status: Dukan.Status
     ): Int
+
+    @Modifying
+    @Query("UPDATE Dukan d SET d.activationStatus = :status, d.deactivationReason = :reason\n WHERE d.id = :dukanId")
```

Comment:
```suggestion
    @Query("UPDATE Dukan d SET d.activationStatus = :status, d.deactivationReason = :reason WHERE d.id = :dukanId")
```
what is this \n I think it's a mistake

Reply:
yeah